shell/components/Tabbed/index.vue

```diff
@@ -86,6 +91,11 @@ export default {
     sortedTabs() {
       return sortBy(this.tabs, ['weight:desc', 'labelDisplay', 'name']);
     },
+
+    // hide tabs based on tab count IF flag is active
+    hideTabs() {
+      return !this.hideSingleTab || (this.hideSingleTab && this.sortedTabs.length > 1);
```

Comment:
Is this logic correct?

In the usage below, we're really using it as showTabs, not hideTabs (in the v-if).

Suggest we leave this as `hideTabs`, but change the v-if below to `!hideTabs`.

I think the logic here can be simply:

`return this.hideSingleTab && this.sortedTabs.length === 1;`

I think that's easier to read?

Reply:
I did the opposite 🤦 . Totally correct @nwmac . Going to change it. Thanks